stacks_queues/queue.py

```diff
@@ -15,47 +15,94 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+
+        if (self.front == (self.rear + 1)%(len(self.store))):
```

Comment:
Rather than calling the `len()` function on `self.store`, we can take advantage of our `buffer_size` attribute

```suggestion
        if (self.front == (self.rear + 1)%self.buffer_size):
```